sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -745,7 +768,21 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
   ): Promise<HttpOperationResponse> {
     const webResource: WebResource = new WebResource(this.getUrl(name), "GET");
 
-    return executeAtomXmlOperation(this, webResource, serializer);
+    const response = await executeAtomXmlOperation(this, webResource, serializer);
+    if (
+      response.parsedBody == undefined ||
+      (Array.isArray(response.parsedBody) && response.parsedBody.length == 0)
+    ) {
+      const err = new RestError(
+        `The messaging entity "${name}" being requested cannot be found.`,
+        "404",
+        404,
```

Comment:
The second parameter above is "code" and the third is "statusCode"
While I understanding using 404 for "statusCode", is using the same for "code" as well right?
From what I understood, `code` was meant to be a string and so, I would imagine this to be something like `ResourceNotFound`

@bterlson to confirm

Reply:
this is the error code returned by service for empty gets on subscription and rules

Reply:
Apparently a `code` of 404 comes back from the service (in addition to the status code of 404), so rather than invent a better status code, I'd say probably best to leave as-is to keep close to the service?